tools/deploy_wiremock.py

```diff
@@ -11,18 +20,103 @@ def main():
     deploy_wiremock(deploy_options)
     log.info('Wiremock deployment completed')
 
-def deploy_wiremock(deploy_options):
+
+def deploy_wiremock(deploy_options: Namespace):
     docs = utils.load_yaml_file_docs('deploy/wiremock/wiremock-deployment.yaml')
     utils.set_namespace_in_yaml_docs(docs, deploy_options.namespace)
+        
     dst_file = utils.dump_yaml_file_docs('build/wiremock-deployment.yaml', docs)
 
+    if deploy_options.target == "kind":
+        utils.override_service_type_definition_and_node_port(
+            internal_definitions_path="build/wiremock-deployment.yaml",
+            internal_target_definitions_path="build/wiremock-deployment.yaml",
+            new_type="NodePort",
+            node_port=30002
+        )
+
     log.info('Deploying %s', dst_file)
     utils.apply(
         target=deploy_options.target,
         namespace=deploy_options.namespace,
         file=dst_file
     )
 
+    if deploy_options.target == "kind":
+        hostname = os.environ.get('IP', 'localhost')
+        port="8070"
+
+    else:
+        log.info("Waiting for Wiremock service to be ready...")
```

Comment:
Maybe we can comment why we need to wait for wiremock service if not kind target? I take it it's because we're waiting for the load balancer to be provisioned, right?